docs/source/installation.md

```diff
@@ -16,183 +16,182 @@ limitations under the License.
 ```
 
-to check 🤗 Transformers is properly installed.
-
 ## Editable install
 
-If you want to constantly use the bleeding edge `master` version of the source code, or if you want to contribute to the library and need to test the changes in the code you're making, you will need an editable install. This is done by cloning the repository and installing with the following commands:
+You will need an editable install if you'd like to:
+
+* Use the `master` version of the source code.
+* Contribute to `transformers` and need to test changes in the code.
+
+Clone the repository and install 🤗 Transformers with the following command:
 
-``` bash
+```bash
 git clone https://github.com/huggingface/transformers.git
 cd transformers
 pip install -e .
 ```
 
-This command performs a magical link between the folder you cloned the repository to and your python library paths, and it'll look inside this folder in addition to the normal library-wide paths. So if normally your python packages get installed into:
-```
-~/anaconda3/envs/main/lib/python3.7/site-packages/
-```
-now this editable install will reside where you clone the folder to, e.g. `~/transformers/` and python will search it too.
+This command links the folder you cloned the repository to and your Python library paths. Python will now look inside this folder in addition to the normal library paths. For example, if your Python packages are typically installed in `~/anaconda3/envs/main/lib/python3.7/site-packages/`, Python will also search the folder you cloned to: `~/transformers/`.
 
-Do note that you have to keep that `transformers` folder around and not delete it to continue using the  `transformers` library.
+<Tip warning={true}>
+You must keep the `transformers` folder if you want to keep using the library.
+</Tip>
 
-Now, let's get to the real benefit of this installation approach. Say, you saw some new feature has been just committed into `master`. If you have already performed all the steps above, to update your transformers to include all the latest commits, all you need to do is to `cd` into that cloned repository folder and update the clone to the latest version:
+Now you can easily update your clone to the latest version of `transformers` with the following commands:
 
-```
+```bash
 cd ~/transformers/
 git pull
 ```
 
-There is nothing else to do. Your python environment will find the bleeding edge version of `transformers` on the next run.
+Your Python environment will find the `master` version of `transformers` on the next run.
 
+## Installation with conda
 
-## With conda
+Install from the conda channel `huggingface`:
 
-Since Transformers version v4.0.0, we now have a conda channel: `huggingface`.
-
-🤗 Transformers can be installed using conda as follows:
-
-```
+```bash
 conda install -c huggingface transformers
 ```
 
-Follow the installation pages of TensorFlow, PyTorch or Flax to see how to install them with conda.
```

Comment:
I would keep a mention of that.

Reply:
Since we linked to the installation pages for the various libraries at the top, do you think it'd be safe to assume users will have already installed `torch` or `tf` with either conda or pip? This way, we don't have to point users to the installation pages again. Or we could also drop this in as a **Tip** as a reminder?